docs/UsingPerkStore.rst

```diff
@@ -0,0 +1,106 @@
+.. _UsingPerkStore:
+
+####################
+Using the Perkstore
+####################
+
+The perkstore is an online store available in the platform. 
+You are able to buy the available products or sell goods or services using the cauris as a currency.
+
+How to access to Perkstore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access to perkstore from the left navigation. By clicking on the appropriate icon, you will display all the available products in the store.
+You can see your wallet balance in the toolbar as well as a search bar and a filter button that allow you to look for a specific product.
+
+|image0|
+
+You can filter the display of articles by:
+- Disable products
+- Soldout products
+- Created by me
+
+
+You can see more details about a product by clicking on its title. You can buy the product from this view as well as the perkstore view.
+
+|image1|
+
+
+How to buy a product and display your orders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To buy a product you are interested in, you have to click on the appropriate button |image2| then specify the number of the items to purchase as well as your wallet password if it is not saved in the browser you are using.
+
+|image3|
+
+After buying the product, You recieve a notification when your order is totally or partially delivered.
+
+|image4|
+
+In the perkstore view a new mention "1 pending" is available for you in the purchased product's card 
+
+|image5|
+
+You can see your order details for this product by clicking in this button.
+
+|image6|
+
+To see all orders you can click on "My orders" button from the toolbar of perkstore.
+
+|image7|
+
+|image8|
+
+
+How to create a product
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are allowed to create products from perkstore, you will see the creation card as the first card displayed in the interface. 
+
+|image9|
+
+By clicking on it, the creation form is displayed.
+
+|image10|
+
+You have to fill in the mandatory fields which are:
+- The name of the product, 
+- The selling price: The product's price in Cauris
+- The merchant wallet: The user's or space's wallet that will ecieve the funds when the product is purchased, and
+- The product editor: This field is automatically pre-loaded with the current user account and you have the possibilityto change this.
```

Comment:
```suggestion
- The product editor: This field is automatically pre-loaded with your user account but you can change it if your want to designate another beneficiary for the product sales revenues


.
```

Reply:
The product editor has the right to edit the product but doesn't receive funds, the funds are sent to the merchant wallet